pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -31,10 +31,16 @@ import (
 type Code int
 
 // NodeScoreList declares a list of nodes and their scores.
-type NodeScoreList []int
+type NodeScoreList []NodeScore
 
-// PluginToNodeScoreMap declares a map from plugin name to its NodeScoreList.
-type PluginToNodeScoreMap map[string]NodeScoreList
+// NodeScore is a struct with node name and score.
+type NodeScore struct {
+	Name  string
+	Score int
+}
+
+// PluginToNodeScoreList declares a map from plugin name to its NodeScoreList.
+type PluginToNodeScoreList map[string]NodeScoreList
```

Comment:
`PluginToNodeScoreList` is a bit confusing. When I saw it in the code above and before seeing the declaration, I thought it was a list. I would suggest using `PluginToNodeScores` to remove the confusion.

Reply:
Done